values.yaml

```diff
@@ -1,5 +1,7 @@
-helperImage: "ghcr.io/private-attribution/ipa/ipa-helper:berkhoff-sd-mt-ra"
-externalPort: 443
-port: 3000
-rustLog: "debug,h2=error"
+helperImage: "ghcr.io/private-attribution/ipa/ipa-helper:main"
+rcImage: "ghcr.io/private-attribution/ipa/rc:main"
+mpcPort: 443
+shardPort: 1443
+shardCount: 1
+rustLog: "debug,tower_http=error,hyper_util=error,rustls=error,h2=error"
```

Comment:
is there a way to configure this at runtime? I worry about debug logging slowing down IPA execution significantly.

Reply:
Yeah, this is just the initial, overridable value